README.md

```diff
@@ -71,6 +72,10 @@ In order to run this project you need:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231857827-623fd002-1b2a-452e-a17d-2d6746c6d205.png)

**[OPTIONAL]**

- [ ] Since you are not using any Framework so you can delete the information highlighted in the image above to make your readme file clean 👍